src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -29,6 +29,7 @@ protocol CoronaWarnAppDelegate: AnyObject {
 	var exposureManager: ExposureManager { get }
 	var taskScheduler: ENATaskScheduler { get }
 	var serverEnvironment: ServerEnvironment { get }
+	var warnOthers: OthersWarnable { get }
```

Comment:
I'm wondering if it really makes sense to put this in such a central place. Couldn't you just initialize it wherever needed with the same store?

Reply:
I will take this as refactoring for the next sprint.